examples/cndpfwd/main.c

```diff
@@ -160,24 +158,36 @@ _l3fwd_test(jcfg_lport_t *lport, struct fwd_info *fwd)
     if (n_pkts == PKTDEV_ADMIN_STATE_DOWN)
         return -1;
 
+    struct ether_addr *eaddr = malloc (n_pkts * sizeof(struct ether_addr));
+    uint16_t *tx_port = malloc (n_pkts * sizeof(uint16_t));
+    uint32_t *ip_addr = malloc (n_pkts * sizeof(uint32_t));
+
+    if (eaddr == NULL || tx_port == NULL || ip_addr) {
```

Comment:
The coding style for CNDP does not allow for single line for, if, while, do, ... statements to have {} around the line.

The CONTRIBUTING.md file should help with coding style.